### PR TITLE
docs: add version matchups for default

### PR DESCRIPTION
**Pull request by shadowspawn**
Add versions which include support for `default` in input configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,10 +234,11 @@ It is exceedingly difficult to provide an API which would both be friendly to th
 
 ## Version Matchups
 
-| Node.js | @pkgjs/parseArgs |
-| -- | -- |
+| Node.js | @pkgjs/parseArgs | Changes |
+| -- | -- | -- |
 | [v18.3.0](https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig) | [v0.9.1](https://github.com/pkgjs/parseargs/tree/v0.9.1#utilparseargsconfig) |
-| [v16.17.0](https://nodejs.org/dist/latest-v16.x/docs/api/util.html#utilparseargsconfig), [v18.7.0](https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig) | [0.10.0](https://github.com/pkgjs/parseargs/tree/v0.10.0#utilparseargsconfig) |
+| [v16.17.0](https://nodejs.org/dist/latest-v16.x/docs/api/util.html#utilparseargsconfig), [v18.7.0](https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig) | [0.10.0](https://github.com/pkgjs/parseargs/tree/v0.10.0#utilparseargsconfig) | tokens |
```
Comment (Eomm) [inline README.md:240]:
```suggestion
| [v16.17.0](https://nodejs.org/dist/latest-v16.x/docs/api/util.html#utilparseargsconfig), [v18.7.0](https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig) | [0.10.0](https://github.com/pkgjs/parseargs/tree/v0.10.0#utilparseargsconfig) | add support for returning detailed parse information using `tokens` in input config and returned properties. |
```

File: README.md [hunk 2]
```diff
@@ -234,10 +234,11 @@ It is exceedingly difficult to provide an API which would both be friendly to th
...
+| [v18.11.0](https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig) | [0.11.0](https://github.com/pkgjs/parseargs/tree/v0.11.0#utilparseargsconfig) | default |
```
Comment (Eomm) [inline README.md:241]:
```suggestion
| [v18.11.0](https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig) | [0.11.0](https://github.com/pkgjs/parseargs/tree/v0.11.0#utilparseargsconfig) | add new `default` field option |
```

**Review by Eomm — COMMENTED** [review]
Suggesting the same string in the node.js website https://nodejs.org/api/util.html#utilparseargsconfig

File: README.md (hunk 2 above)
Comment (shadowspawn) [reply README.md:241]: Good idea

Comment (shadowspawn) [reply README.md]: I see different history comment in Node.js docs for this one, copied and pasted from web page.

**Review by Eomm — APPROVED** [review]

**Review by aaronccasanova — APPROVED** [review]

**Pull request merged**